docling_eval/evaluators/bbox_text_evaluator.py

```diff
@@ -100,12 +100,10 @@ def __call__(
         self,
         ds_path: Path,
         split: str = "test",
-        external_predictions_path: Optional[Path] = None,
+        ext_docdoc_loader: Optional[ExternalDoclingDocumentLoader] = None,
```

Comment:
Couple naming alternatives, as "docdoc" sounds odd & doesn't align with PEP-8:
- `ext_doc_loader`, or
- `ext_dl_doc_loader` if we really need to differentiate (`dl_doc` is also used in other contexts for this purpose)

Comment applies to other occurrences too.